SU2_CFD/include/output/CFlowCompOutput.hpp

```diff
@@ -0,0 +1,131 @@
+/*!
+ * \file CFlowCompOutput.hpp
+ * \brief  Headers of the compressible flow output.
+ * \author R. Sanchez, T. Albring.
+ * \version 6.2.0 "Falcon"
+ *
+ * The current SU2 release has been coordinated by the
+ * SU2 International Developers Society <www.su2devsociety.org>
+ * with selected contributions from the open-source community.
+ *
+ * The main research teams contributing to the current release are:
+ *  - Prof. Juan J. Alonso's group at Stanford University.
+ *  - Prof. Piero Colonna's group at Delft University of Technology.
+ *  - Prof. Nicolas R. Gauger's group at Kaiserslautern University of Technology.
+ *  - Prof. Alberto Guardone's group at Polytechnic University of Milan.
+ *  - Prof. Rafael Palacios' group at Imperial College London.
+ *  - Prof. Vincent Terrapon's group at the University of Liege.
+ *  - Prof. Edwin van der Weide's group at the University of Twente.
+ *  - Lab. of New Concepts in Aeronautics at Tech. Institute of Aeronautics.
+ *
+ * Copyright 2012-2019, Francisco D. Palacios, Thomas D. Economon,
+ *                      Tim Albring, and the SU2 contributors.
+ *
+ * SU2 is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * SU2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "CFlowOutput.hpp"
+
+class CVariable;
+
+/*! \class CFlowCompOutput
+ *  \brief Output class for compressible flow problems.
+ *  \author R. Sanchez, T. Albring.
+ *  \date May 30, 2018.
+ */
+class CFlowCompOutput final: public CFlowOutput {
+private:
+  
+  unsigned short turb_model; //!< Kind of turbulence model
```

Comment:
If this, and similar variables are assigned Enum values it would be better to use the appropriate type.

Reply:
That would require also to change the corresponding config routine, so I'll leave it like that for the moment. But in general you are right, we should keep that in mind.